webapp/components/survey/CategoryDetails/CategoryDetails.js

```diff
@@ -59,7 +60,7 @@ const CategoryDetails = (props) => {
             <UploadButton
               label={i18n.t('common.csvImport')}
               accept=".csv"
-              onChange={(files) => dispatch(Actions.uploadCategory(Category.getUuid(category), files[0]))}
+              onChange={(files) => dispatch(CategoryActions.uploadCategory(Category.getUuid(category), files[0]))}
```

Comment:
shall we move to actions?

Reply:
this will be done in another pull request related to the category import ;)